.github/workflows/cmake.yml

```diff
@@ -50,6 +69,8 @@ jobs:
       - name: Configure (macOS)
         shell: bash
         run: |
+             set -e
+             export PKG_CONFIG_PATH=/usr/local/opt/icu4c/lib/pkgconfig:/usr/local/opt/libarchive/lib/pkgconfig:/usr/local/opt/libffi/lib/pkgconfig:$PKG_CONFIG_PATH
              mkdir build
```

Comment:
A more general solution would use `$(brew --prefix)` instead of `/usr/local`: Homebrew for macOS on Arm64 suggests a different prefix `/opt/homebrew`, so `/usr/local` would not work there.